File: public/phase-2-20160620022203/C/changing-root-of-tree.c
// Name : Changing Root of tree
#include <stdio.h>
int main() {
	// input:
	// n :  the amount of nodes in the tree
	// r1 : index of the old root
	// r2 : e index of the new one
	// old_tree :  old representation of the tree
	int n, r1, r2, old_tree[104], new_tree[104]={2,3}, index;
	scanf("%d %d %d", &n, &r1, &r2);
	for(index = 0; index< (n-1); index++) {
		scanf("%d", &old_tree[index]);
	}


	// write your code here
	// store your results in `new_tree`

	// output
	for(index = 0; index< (n-1); index++) {
		printf("%d ",new_tree[index]);
	}
	return 0;
}

File: public/phase-2-20160620022203/PYTHON/changing-root-of-tree.py
#!/usr/bin/python
# Name : Changing Root of tree
# input:
# n :  the amount of nodes in the tree
# r1 : index of the old root
# r2 : e index of the new one
# old_tree :  old representation of the tree
elements = raw_input().strip().split(' ')
n = int(elements[0])
r1 = int(elements[1])
r2 = int(elements[2])
old_tree_elements = raw_input().strip().split(' ')
old_tree = map(int, old_tree_elements)


# write your code here
# store your results in `new_tree`

# output

# Dummy Data
new_tree = {2,3}

print(" ".join(str(e) for e in new_tree))